File: frontend/core/src/com/sopra/ntts/model/NetworkStandard/Messages/HelloMessage.java
package com.sopra.ntts.model.NetworkStandard.Messages;

import com.sopra.ntts.model.NetworkStandard.DataTypes.RoleEnum;
import com.sopra.ntts.model.NetworkStandard.MessageContainer.MessageContainer;
import com.sopra.ntts.model.NetworkStandard.MessageTypeEnum.MessageTypeEnum;

import java.util.Date;
import java.util.UUID;

/**
 * Initial message when connection is established, send by the server.
 *
 * @author Standardisierungsdokument & Sedat Qaja
 */
public class HelloMessage extends MessageContainer {
  public String name;
  public RoleEnum role;

  public HelloMessage(UUID playerId, MessageTypeEnum type, Date creationDate, String debugMessage, String name, RoleEnum role) {
    super(playerId, MessageTypeEnum.HELLO, creationDate, debugMessage);
    this.name = name;
    this.role = role;
  }


}

File: dummy-ai-client/NetworkStandard/Messages/hello_message.py
from NetworkStandard.MessageContainer.message_container import MessageContainer
from NetworkStandard.MessageTypeEnum.messagetype_enum import MessageTypeEnum
from NetworkStandard.DataTypes.role_enum import RoleEnum

import uuid


class HelloMessage(MessageContainer):

    def __init__(self, player_Id, type_enum, creation_date, debug_message, name,
                 role):
        super().__init__(player_Id, type_enum, creation_date, debug_message)
        self.name = name
        self.role = role
